modules/compose/templates/docker-compose-container.yml.erb

```diff
@@ -0,0 +1,150 @@
+# ###################################
+# This file is generated by puppet
+# PLEASE DON'T MODIFY BY HAND
+# ###################################
+version: '2'
+services:
+
+
+########################
```

Comment:
The format of the compose file in a container is not suitable to run on the host.  We now generate a second file that can be launched directly on the host.